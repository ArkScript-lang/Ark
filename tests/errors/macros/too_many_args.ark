!{foo (a b c)
    (+ a b c)}

(foo 1 2 3 4)