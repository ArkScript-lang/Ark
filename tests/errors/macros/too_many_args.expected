Macro `foo' got 4 argument(s) but needed 3